docs/manual/macros.md

```diff
@@ -22,14 +22,14 @@ To define a macro use:
 	%define <name>[(opts)] <body>
 ```
 
-All whitespace surrounding \<body\> is removed.  Name may be composed
-of alphanumeric characters, and the character `_' and must be at least
-3 characters in length. A macro without an (opts) field is "simple" in that
-only recursive macro expansion is performed. A parameterized macro contains
-an (opts) field. "-" as opts disables all option processing, otherwise
-the opts (i.e. string between parentheses) are passed
+All whitespace surrounding `<body>` is removed.  Name may be composed
+of alphanumeric characters and the character `_`, and must be at least
+3 characters in length. A macro without an `(opts)` field is "simple"
+in that only recursive macro expansion is performed. A parameterized
+macro contains an `(opts)` field. `-` as opts disables all option
+processing, otherwise the opts (i.e. string between parentheses) are passed
```

Comment:
I snuck in a grammar change here that really should've been a separate commit, moving the comma in this sentence:

>  Name may be composed of alphanumeric characters, and the character `_` and must be at least 3 characters in length.

to separate the two clauses:

>  Name may be composed of alphanumeric characters and the character `_`, and must be at least 3 characters in length.